editoast/editoast_schemas/src/infra/railjson.rs

```diff
@@ -16,21 +17,38 @@ use super::TrackSection;
 
 pub const RAILJSON_VERSION: &str = "3.4.11";
 
-#[derive(Deserialize, Derivative, Serialize, Clone, Debug)]
+editoast_common::schemas! {
+    RailJson,
+}
+
+/// The `RailJson` structure represents the data model for the RailJSON format.
```

Comment:
```suggestion
/// An infrastructure description in the RailJson format
```
 
I feel like the doc of each object could be better placed on their respective schemas and just put a `/// A list of XYZ` for each vec<> here. But this can be done in another PR :)